src/fuse_scan.cpp

```diff
@@ -113,11 +209,11 @@ class FusedScan {
                 scan_fuse.ranges[i]= 40.0;
             }
             for (long int i = 0; i < cloud_fuse.points.size(); i++){
+                Point p(cloud_fuse.points[i].x, cloud_fuse.points[i].y);
```

Comment:
give a better name, may be lidar_pt , or cloud_pt or something else...definitely not a single letter